rgw/admin/bucket.go

```diff
@@ -19,15 +20,16 @@ type Bucket struct {
 		DataExtraPool string `json:"data_extra_pool"`
 		IndexPool     string `json:"index_pool"`
 	} `json:"explicit_placement"`
-	ID        string `json:"id"`
-	Marker    string `json:"marker"`
-	IndexType string `json:"index_type"`
-	Owner     string `json:"owner"`
-	Ver       string `json:"ver"`
-	MasterVer string `json:"master_ver"`
-	Mtime     string `json:"mtime"`
-	MaxMarker string `json:"max_marker"`
-	Usage     struct {
+	ID           string     `json:"id"`
+	Marker       string     `json:"marker"`
+	IndexType    string     `json:"index_type"`
+	Owner        string     `json:"owner"`
+	Ver          string     `json:"ver"`
+	MasterVer    string     `json:"master_ver"`
+	Mtime        string     `json:"mtime"`
+	CreationTime *time.Time `json:"creation_time"`
```

Comment:
Is this clearly defined format putting time.Time into a Json property?

Reply:
Go uses [RFC 3339](https://pkg.go.dev/time#Time.UnmarshalJSON).

Reply:
Good news. Even though this doesn't change the fucntion signatures I think it would be good to have a unit test for this change. It will assure us that the Go JSON support is compatible with the output of the RGW system.  Thanks!

Reply:
Is there a way to run the tests in docker similar to `make test-docker` but only run the tests in the `rgw/admin` folder? The equivalent of running `go test -run ^TestListBucketsWithStat$ github.com/ceph/go-ceph/rgw/admin`

Reply:
You can make use of  `ENTRYPOINT_ARGS` as follows:
```
make test-docker ENTRYPOINT_ARGS="--test-pkg=rgw/admin --test-run=TestRadosGWTestSuite/TestListBucketsWithStat"
```